rel/run.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
```

Comment:
One last question: Why is this changed?

Reply:
personal preference: because i know the absolute path of the current environment (base alpine docker image) and don't want to search inside $PATH.